ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -384,15 +384,6 @@ class ServicesHomeScreen extends React.Component<Props, State> {
         <Text>{I18n.t("services.loading.subtitle")}</Text>
         {/* TODO: remove this component after the resolution of https://www.pivotaltracker.com/story/show/172431153 */}
         <View spacer={true} extralarge={true} />
```

Comment:
```suggestion
```
maybe also these line are useless now?

Reply:
Yeah, there are some lines that are useless and could be removed, but I preferred to ignore to avoid nasty surprises, so I proceed with the elimination of this line

Reply:
Fixed as suggested ✅